### DISCUSSION TITLE
How Do I Delete My Profile

### DISCUSSION THREAD

**Discussion by KatieOT** [community/community#46921] [New to GitHub]
I came to the wrong site. I don't know anything about coding or anything else in here. So, would someone please tell me how to delete my account. I would love to come back when I know what I'm doing, LOL. Thank you for any help you can provide to me. KatieOT

**Comment by jge162** [comment]
Here are the instructions for deleting your account [link](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account)

**Comment by Juanescacha** [comment]
its quite simple as @jge162 give you the doc link
you just go through this simple step by step
go to settings
![image](https://user-images.githubusercontent.com/6394715/218212436-0081e503-4da2-4e44-8bd3-1a6735e3ade5.png)
choose account
![image](https://user-images.githubusercontent.com/6394715/218212641-3ee4d77f-c970-49a8-af3e-8003caa2b637.png)
go to bottom part
![image](https://user-images.githubusercontent.com/6394715/218212942-1ae1c296-deac-49c3-8a88-f7b601597567.png)
and follow this last instructions
![image](https://user-images.githubusercontent.com/6394715/218213067-0978f5bd-2eab-42da-b7e3-ebd28fbd445b.png)

**Comment by mojidda** [comment]
Am a beginner, and every time I get to git add I always receive an error message. How do I learn to use GitHub properly

**Reply by Juanescacha** [reply]
In YT are many simple tutorials/courses to use correctly git and GitHub and if you want more help from the community try to create a new discussion thread
